controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -1,13 +1,40 @@
 from math import ceil
 from referee.consts import MATCH_TIME, TIME_STEP
 from referee.referee import RCJSoccerReferee
+from datetime import datetime
+from pathlib import Path
+
+
+TEAM_YELLOW = "The Yellows"
+TEAM_BLUE = "The Blues"
+
+
+def reflog_path(directory: Path,
+                team_blue: str,
+                team_yellow: str):
+    now_str = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+    team_blue = team_blue.replace(' ', '_')
+    team_yellow = team_yellow.replace(' ', '_')
+
+    # Ensure the directory ecists
+    if not directory.exists():
+        directory.mkdir(parents=True, exist_ok=True)
+
+    p = directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}.json')
```

Comment:
@aquahika I believe this could be of use for you as well, when it comes to generating the filename of the video.

Let's just do it in the next iteration though, not in the first one :)